code/index.html

```diff
@@ -1,64 +1,73 @@
-        alt="Guess Who"
-      />
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project Name</title>
+  <link rel="stylesheet" href="style.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+</head>
 
-      <h1>Does the person have</h1>
+<body>
+  <aside class="question-section">
+    <button id="restart" class="outlined-button">RESTART</button>
 
-      <select id="questions">
-        <optgroup label="hair color">
-          <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
-        </optgroup>
-        <optgroup label="eye color">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
-        </optgroup>
-        <optgroup label="accessories">
-          <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
-        </optgroup>
-        <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
-        </optgroup>
-      </select>
+    <img class="guess-who-icon" src="images/guess-who-bubble.png" alt="Guess Who" />
 
-      <button id="filter" class="filled-button">FIND OUT</button>
-    </aside>
+    <h1>Does the person have</h1>
 
-    <section class="board-wrapper">
-      <main class="game-board" id="board"></main>
-    </section>
+    <select id="questions">
+      <optgroup label="hair color">
+        <option value="hidden">hidden hair</option>
+        <option value="brown">brown hair</option>
+        <option value="yellow">yellow hair</option>
+        <option value="grey">grey hair</option>
+        <option value="black">black hair</option>
+        <option value="orange">orange hair</option>
+        <option value="purple">purple hair</option>
+        <option value="white">white hair</option>
+        <!-- Add the rest of hair colors as options  -->
+      </optgroup>
+      <optgroup label="eye color">
+        <option value="hidden">hidden eyes</option>
+        <option value="blue">blue eyes</option>
+        <option value="brown">brown eyes</option>
+        <option value="green">green eyes</option>
+        <!-- Add the rest of eye colors as options  -->
+      </optgroup>
+      <optgroup label="accessories">
+        <option value="glasses">glasses</option>
+        <option value="hat">hat</option>
+        <option value="necklace">necklace</option>
+        <option value="earrings">earrings</option>
+        <!-- Add the other accessory option here. (hat)  -->
+      </optgroup>
+      <optgroup label="other">
+        <option value="smoker">a smoking habit</option>
+      </optgroup>
+    </select>
```

Comment:
Maybe it would be a nice think to generate long lists of options with your Javascript? It is nothing wrong, with this approach, but It makes you type a lot 😸